src/containers/QuoteDisplayerFn.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react'; 
+import Button from '../components/Button/Button.js';
+import Quote from '../components/Quote/Quote.js';
+import { fetchQuote } from '../services/fetchQuote.js';
+
+
+const QuoteDisplayerFn = () => {
+  const [quoteObj, onClick] = useState({
+    character: '', 
+    quote: '',
+    image: ''
+  });
+
+  const handleClick = () => {
+    return fetchQuote()
+      .then(quote => onClick({ quoteObj: quote }));
+  };
+
+  //no useEffect needed
+
+  return (
+    <>
+      <Button onClick={handleClick} />
```

Comment:
and then here, I might recommend passing:
```
<Button handleClick={handleClick} />
```

Reply:
Gotcha, so functional components pass these handlers directly ;)